pandas/_config/config.py

```diff
@@ -820,6 +821,27 @@ def inner(x):
     return inner
 
 
+def is_pos_int(value):
+    """
+    Raises ValueError if type(x) is not equal to None or int, or if x is not
+    positive in the case of a int type.
+
+
+    Parameters
+    ----------
+    value - the value to be checked
```

Comment:
@jreback comment for returns section is lost in diff so re-commenting here but I'm surprised this isn't failing the CI either

@datapythonista do we ignore private docstrings in the validation? Vaguely recall some comment around that but didn't see code to confirm

Reply:
Sorry, shall I add the following?

```
Returns
----------
Nothing 
```

Reply:
@WillAyd yes, we don't validate docstrings of private functions. Ideally we should validate everything, but with all the pending work in the public API that has an impact in the web docs, I don't think we should spend time in fixing the private docstrings yet (and if we don't fix the existing cases, we can't easily validate the new ones).

Reply:
Thanks for the clarification - do you know where that exclusion happens? Didn't see it in code_checks and couldn't find on quick glance in validate_docstrings.py

Reply:
Ups sorry, was going to mention and I forgot. It's not that we exclude them, it's that we get the docstrings to consider from the `.rst` files of the API docs. The code here is a good starting point to understand that: https://github.com/pandas-dev/pandas/blob/master/scripts/validate_docstrings.py#L873

Reply:
@Adam-Klaum these need to be updated to follow the doctoring stander. See https://dev.pandas.io/development/contributing.html#updating-a-pandas-docstring